build.gradle

```diff
@@ -34,8 +35,8 @@ gradlerio.team = "1294"                                   //Your FRC team number
 
 
 dependencies {
-	compile files(System.properties['user.home'] + '/wpilib/java/extracted/library/lib/NetworkTables.jar')
-	compile files(System.properties['user.home'] + '/wpilib/java/extracted/library/lib/WPILib.jar')
+    compile files(System.properties['user.home'] + '/wpilib/java/extracted/library/lib/NetworkTables.jar')
```

Comment:
That being said, I will admit to not demonstrating any discipline in this project yet as to this matter. I had left a mess in this regard.